nexus-sdk-js/tsconfig.json

```diff
@@ -40,14 +40,14 @@
     // "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
 
     /* Module Resolution Options */
-    "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
+    "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
```

Comment:
I'm pretty sure this is an Prettier auto-fix, rather than me doing anything. `tsconfig` is using their own  `.json` stuff where you can put comments in it